<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>increase_credit_dialog</class>
 <widget class="QDialog" name="increase_credit_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="cardName_lable">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Card Holder Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardName_le">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="amount_lable">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="amountSpinBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardNo_le">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>321</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="cardNo_lable">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Card Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cvv_lable">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CVV:</string>
   </property>
  </widget>
  <widget class="QLabel" name="expDate_lable">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Expiration Date:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cvv_le">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="payButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="discardBotton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Discard</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
